ui/apps/platform/src/Containers/Collections/CollectionForm.tsx

```diff
@@ -290,9 +292,38 @@ function CollectionForm({
                                 spaceItems={{ default: 'spaceItemsMd' }}
                                 direction={{ default: 'column' }}
                             >
-                                <div className="pf-u-background-color-100 pf-u-p-lg">
+                                <Flex
+                                    className="pf-u-background-color-100 pf-u-p-lg"
+                                    direction={{ default: 'column' }}
+                                    spaceItems={{ default: 'spaceItemsMd' }}
+                                >
                                     <Title headingLevel="h2">Collection details</Title>
-                                </div>
+                                    <Flex>
+                                        <FlexItem flex={{ default: 'flex_1' }}>
+                                            <FormGroup label="Name" isRequired>
+                                                <TextInput
+                                                    id="name"
+                                                    name="name"
+                                                    value={values.name}
+                                                    validated={errors.name ? 'error' : 'default'}
+                                                    onChange={(_, e) => handleChange(e)}
+                                                    onBlur={handleBlur}
+                                                />
+                                            </FormGroup>
+                                        </FlexItem>
+                                        <FlexItem flex={{ default: 'flex_2' }}>
```

Comment:
Can you please take a picture at a narrower screen width to illustrate responsive behavior?